app/controllers/projects_controller.rb

```diff
@@ -19,9 +19,10 @@ def new
   end
 
   def create
-    @project = Project.new(project_params)
+    @note = Note.new(**note_params, user_id: user_id)
+    @project = Project.new(**project_params, regional_delivery_officer_id: user_id, notes_attributes: [@note.attributes])
```

Comment:
I wonder if it's worth slicing this tidying up of RDO assignment into its own commit? That's a very weak wonder though.